src/js/questionPage.js

```diff
@@ -1,3 +1,24 @@
+// test array
+const questions = [
+	{
+		question:"2+2",
+		answer1:"4",
+		answer2:"3",
+		answer3:"2"
+	},
+	{
+		question:"2+1",
+		answer1:"4",
+		answer2:"3",
+		answer3:"2"
+	},
+]
+
+let questionCounter=0;
+let avaibleQuestions=[];
+let currentQuestion={};
+const answers = Array.from(document.getElementsByClassName("answer__button"));
```

Comment:
We could name it like `answerElements`/`answerNodes` to tell that those are html elements and not just strings